### PR TITLE
create tag put/delete route [#234]

### PR DESCRIPTION
**Pull request by cadecairos**

### CODE REVIEW

**Comment by alicoding** [comment]
@cadecairos code looks good. How can I test this locally?

**Comment by cadecairos** [comment]
check out the makeapi-client PR i assigned you

**Comment by alicoding** [comment]
@cadecairos you meant this one? https://github.com/mozilla/makeapi-client/issues/45 the one you are now assigned to yourself? :P

**Comment by cadecairos** [comment]
>.> mebbe

File: server.js [hunk 1]
```diff
@@ -211,6 +211,30 @@ app.get(
...
+  middleware.canTag,
+  routes.tag
+);
+
+// remove Tag from Make
+app.del(
+  "/api/20130724/make/:id/tag/:tag",
+  Mongo.isDbOnline,
+  middleware.hawkAuth,
+  middleware.setHatchetEventType( "add_tag" ),
```
Comment (alicoding) [inline server.js]: should be delete tag here.

**Comment by alicoding** [comment]
also fix from `credentials.isAdmin = true;` to `credentials.admin = true;` in `MakeAPI/lib/strategy.js`

**Comment by alicoding** [comment]
r+

**Pull request merged**